ci-operator/step-registry/upi/conf/vsphere/zones/upi-conf-vsphere-zones-commands.sh

```diff
@@ -186,6 +186,25 @@ networking:
 EOF
 fi
 
+declare zone_worker_ip
+
+num=$((6 +  $WORKER_REPLICAS))
+for i in $(seq 7 $num); do
+    worker_ip="192.168.${third_octet}.$i"
+    echo $worker_ip
+    zone_worker_ip+="\"$worker_ip\"",
+done
+
+declare zone_master_ip
+
+num=$((3 +  $MASRER_REPLICAS))
```

Comment:
when $MASRER_REPLICAS > 3, the ip will overlap with worker's ip. The worker `num`  should be based on master `num`.